Components/Input/input.html

```diff
@@ -0,0 +1,192 @@
+                <a class="side-bar-items" href="../Responsive-Image/image.html">Image</a>
+                <a class="side-bar-items active" href="../Input/input.html">Input</a>
+                <a class="side-bar-items" href="#">List</a>
+                <a class="side-bar-items" href="#">Toast</a>
+            </section>
+        </div>
+        <div class="wrapper">
+
+            <div class="side-bar">
+
+                <span class="side-bar-title">Get Started</span>
+                <div id="line">
+                    <hr class="hr-sideBar" />
+                </div>
+                <a class="side-bar-items" href="../../Get Started/introduction.html">Introduction</a>
+                <a class="side-bar-items" href="#">Colors</a>
+                <a class="side-bar-items" href="#">Typography</a>
+                <a class="side-bar-items" href="#">Conclusion</a>
+
+                <span class="side-bar-title">Components</span>
+                <div id="line">
+                    <hr class="hr-sideBar" />
+                </div>
+
+                <a class="side-bar-items" href="../Avatar/avatar.html">Avatar</a>
+                <a class="side-bar-items " href="../Alert/alert.html">Alert</a>
+                <a class="side-bar-items " href="../Badge/badge.html">Badge</a>
+                <a class="side-bar-items " href="../Button/button.html">Button</a>
+                <a class="side-bar-items" href="../Cards/card.html">Card</a>
+                <a class="side-bar-items" href="../Responsive-Image//image.html">Image</a>
+                <a class="side-bar-items active" href="../Input/input.html">Input</a>
+
+                <a class="side-bar-items" href="#">List</a>
+                <a class="side-bar-items" href="#">Toast</a>
+
+            </div>
+
+            <div class="content">
+                <span class="title">Inputs</span>
+                <p class="margin-bottom-1 p-content">Inputs are necessary for user interaction with the website. Inputs
+                    are used to take information from the user, and then save the data to the server, accordingly.
+
+                </p>
+                <hr class="margin-bottom-1">
+                <!-- Label Inputs
+ -->
+                <span class="sub-title">Label Inputs
+                </span>
+                <p class="p-content">Label Input has a label present at the top of the input field. They can be
+                    customized to show error error-message, and change color accordingly.
+                </p>
+
+                <div class="input-container">
+
+                    <div class="input-group">
+                        <label class="label">Username *</label>
+                        <input class="input" type="text">
+                    </div>
+
+                    <div class="input-group input-error">
+                        <label class="label">Password *</label>
+                        <input class="input" type="password">
```

Comment:
you can also put required keyword here it will check whether the field is empty or not

Reply:
Required added , Thanks for the review